docs/commerce/storefront/extend_storefront.md

```diff
@@ -0,0 +1,92 @@
+---
+description: Extend Storefront configuration
+edition: commerce
+---
+
+# Extend storefront
+
+## Built-in menus
+
+With the `ibexa\storefront` package come the following built-in menus:
+
+| Item   | Value     | Description |
+|------------|----------|---------|
+| [Breadcrumbs](#breadcrumbs-menu)| - | Renders breadcrumbs for Content Tree Root, Taxonomy Entry, Product, User settings and User settings group | 
+| [Taxonomy](#taxonomy-menu)| - | Renders a menu to change the active currency |               
+| Currency| `currency_menu` | Renders a menu to change the active currency |               
+| Currency| `currency_menu` | Renders a menu to change the active currency |               
+| Language| `langauge_menu` | Renders a menu to change the active language |
+| Region  | `region_menu`  | Renders a menu to change thr active region |
+
+Usage example:
+
+```html_twig
+{% set currrency_menu = knp_menu_get('ibexa_storefront.menu.currency') %}
+
+{{ knp_menu_render(currrency_menu) }}
+```
+
+### Breadcrumbs menu
+
+To add breadcrumbs menu to the product, you need to use an event subscriber.
+
+Create an event subscriber in the `src/EventSubscriber/BreadcrumbsMenuSubscriber.php`:
```

Comment:
```suggestion
Create an event subscriber in `src/EventSubscriber/BreadcrumbsMenuSubscriber.php`:
```

Reply:
Actually, it's not that way. Too add the menu you need only the templates below. You need the subscriber to modify the items in the menu